### ISSUE TITLE
[JENKINS-70209] Letter spacing should not be negative (regression in 2.340 + 2.350)

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-06-a2#5879]
<p>Negative letter spacing should be avoided. It greatly decreases legibility and accessibility.</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/59618/59618_image-2022-12-05-16-54-00-900.png" style="border: 0px solid black" /></span></p>

**Attachments:**

- [image-2022-12-05-16-54-00-900.png](https://issues.jenkins.io/secure/attachment/59618/image-2022-12-05-16-54-00-900.png)

---
<details><summary><i>Originally reported by <a title="J" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=gadzy">gadzy</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-70209" target="_blank">Letter spacing should not be negative (regression in 2.340 + 2.350)</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a title="J" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=gadzy">gadzy</a>
<li><b>reported by</b>: gadzy
<li><b>status</b>: Closed
<li><b>priority</b>: Minor
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2023-01-01T19:49:12+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 2025-11-06
</ul></i>
</details>
